### ISSUE TITLE
Documentation updates

### ISSUE THREAD

**Issue by krigar1184** [LucidtechAI/las-docs#8]
At the moment documentation is generated differently for different repositories and merged manually into common repo (las-docs), which makes supporting it difficult. Thus I suggest some improvements:
- choosing one structure local documentation will follow (and preferably using the same tool to generate it)
- using a tool (something rsync-like) to automate merging of the documentation into the common repo)

**Comment by augustak** [comment]
I agree that we need to automate this better. What do you think about having a git action that pushes to las-docs whenever a PR is accepted into master? The flow could be something along the lines of:

PR into master in las-sdk-java
Git action in las-sdk-java that does the following
make docs
push to las-docs (beta branch) as new version of java docs by looking at current version in the master branch

Then we manually make a PR from beta to master branch in las-docs from time to time, but "beta" will always have the updated docs

Alternatively we can have a git hook in las-docs that listens to changes in master branch of the different repos and does the same job

make docs
etc

What do you think?

**Comment by krigar1184** [comment]
![docs-workflow](https://user-images.githubusercontent.com/10408234/82808279-f5d05200-9e89-11ea-86db-3753334004b0.png)
